src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -36,7 +37,7 @@ public class ReleaseServiceTests
         private readonly Guid _userId = Guid.NewGuid();
 
         [Fact]
-        public async void CreateReleaseNoTemplate()
+        public async Task CreateReleaseNoTemplate()
```

Comment:
Think return types should be kept as void. I think I've seen the IDEA test runner be a bit funny with running methods with `Task` return types.

Reply:
Are you sure it's not the other way round because there's lots of articles talking about why we avoid `async void` including in unit tests.

Reply:
Fairly sure, but it could have just been IDEA being funny for no particular reason at the time. If you want to keep it like this, it's fine and if I run into problems next time in the area I'll just change them back.

Reply:
Ok, thanks, let me know if you run into some problems! 👍